libcsm/__init__.py

```diff
@@ -0,0 +1,21 @@
+#  MIT License
```

Comment:
I like `libcsm`

What if we wanted a go module by the same name?  Should we further distinguish between the two in the directory structure since we have one for `sh`?  This would leave it open for other languages, too

Reply:
I debated adding Go here for a while, the issue with Go is that if we provide a Go module would actually provide a go binary? That question made me think that if we did add Go, it would be an importable module without a binary, in which case we'd just add a `go/` subdirectory.

If we built a binary here, I'd be more concerned with naming. In the future if we do build a binary, we can still move it to a sub directory and then update the spec file to build a subpackage called `libcsm-go`.

Reply:
Solid copy.